splunk_eventgen/version.json

```diff
@@ -1 +0,0 @@
-{"version": "7.1.0"}
```

Comment:
The `version.json` is now deleted.  We use the version in `pyproject.toml` as single source.